test/games/strategy/engine/framework/map/download/DownloadFileParserTest.java

```diff
@@ -22,53 +19,53 @@
   public void testParseMap() {
     final ByteArrayInputStream inputStream = new ByteArrayInputStream(buildTestXml());
     List<DownloadFileDescription> games = DownloadFileParser.parse(inputStream);
-    assertThat(games.size(), is(4));
+    assertEquals(4, games.size());
     DownloadFileDescription desc = games.get(0);
-    assertThat(desc.getUrl(), is("http://example.com/games/game.zip"));
-    assertThat(desc.getDescription(), Matchers.containsString("Some notes"));
-    assertThat(desc.getMapName(), is(GAME_NAME));
+    assertEquals(desc.getUrl(), "http://example.com/games/game.zip");
```

Comment:
Can you do a partial revert to keep the hamcrest assertions?

Let me know if you'll have trouble doing that, I could potentially help. The git commands I see for this:
``` 
git reset --soft HEAD^
git reset .
git checkout ... < files to revert >
```
Also, if not mentioned already, for Git commands, I very much recommend this tool: https://github.com/ndbroadbent/scm_breeze, let's you reference files by number, so you could do this:
```
git status
git checkout 1 2 3-5  ## rather than listing files out by name
```